modules/adding-ibmz-lpar-agent.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_with_agent_based_installer/prepare-pxe-infra-agent.adoc
+// * installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="adding-ibmz-lpar-agents_{context}"]
+= Adding {ibm-z-title} Logical Partition (LPAR) as agents
+
+With {ocp-short} 4.16, you can add the Logical Partition (LPAR) on IBM Z or IBM LinuxONE as a compute node.
+
+. Create a boot parameter file for the agents.
++
+.Sample parameter file:
+[source,yaml]
+----
+rd.neednet=1 cio_ignore=all,!condev \
+console=ttysclp0 \
+ignition.firstboot ignition.platform.id=metal
+coreos.live.rootfs_url=http://<http_server>/rhcos-<version>-live-rootfs.<architecture>.img \// <1>
+coreos.inst.persistent-kargs=console=ttysclp0
+ip=<ip>::<gateway>:<netmask>:<hostname>::none nameserver=<dns> \// <2>
+rd.znet=qeth,<network_adaptor_range>,layer2=1
+rd.<disk_type>=<adapter> \// <3>
+zfcp.allow_lun_scan=0
+ai.ip_cfg_override=1 \// <4>
+random.trust_cpu=on rd.luks.options=discard
+----
+<1> For the `coreos.live.rootfs_url` artifact, specify the matching `rootfs` artifact for the `kernel` and `initramfs` that you are starting. Only HTTP and HTTPS protocols are supported.
+<2> For the `ip` parameter, manually assign the IP address, as described in _Installing a cluster with z/VM on IBM Z and IBM LinuxONE_. 
+<3> For installations on DASD-type disks, use `rd.dasd` to specify the DASD where Red Hat Enterprise Linux CoreOS (RHCOS) is to be installed. For installations on FCP-type disks, use `rd.zfcp=<adapter>,<wwpn>,<lun>` to specify the FCP disk where RHCOS is to be installed.
+<4> Specify this parameter when you use an Open Systems Adapter (OSA) or HiperSockets.
+
+. Generate the `.ins` and `initrd.img.addrsize` files.
++
+The `.ins` file is a special file that includes installation data and is present on the FTP server. It can be accessed from the HMC system.
++
+This file contains details such as mapping of the location of installation data on the disk or FTP server, the memory locations where the data is to be copied.
++
+*Note*: In {ocp-short} 4.16, the `.ins` file and 'initrd.img.addrsize' are not automatically generated as part of boot-artifacts from the installer. You must manually generate these files.
+
+.. Run the following commands to get the size of the `kernel` and `initrd`: 
++
+[source,yaml]
+----
+KERNEL_IMG_PATH='./kernel.img'
+INITRD_IMG_PATH='./initrd.img'
+CMDLINE_PATH='./generic.prm'
+kernel_size=$(stat -c%s $KERNEL_IMG_PATH )
+initrd_size=$(stat -c%s $INITRD_IMG_PATH)
+----
+.. Round the `kernel` size up to the next MiB boundary. This value  is the starting address of `initrd.img`.
++
+[source,bash]
```

Comment:
```suggestion
[source,terminal]
```
openshift-docs guidelines are different than ACM. source,bash is only used for bash scripts see: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output